docs/actions/screengrab.md

```diff
@@ -152,6 +152,91 @@ If you're having trouble getting your device unlocked and the screen activated t
 
 You can use [QuickDemo](https://github.com/PSPDFKit-labs/QuickDemo) to clean up the status bar for your screenshots.
 
+# Advanced _screengrab_
+
+<details>
+<summary>Launch Arguments</summary>
+
+You can provide additional arguments to your testcases on launch. These strings will be available in your tests through `InstrumentationRegistry.getArguments()`.
+
+```ruby
+launch_arguments([
+  "username hjanuschka",
+  "build_number 201"
+])
```

Comment:
@dustinrwh Try changing this too...
```rb
supply(
  launch_arguments: {
    "username hjanuschka",
    "build_number 201"
  }
)
```

The tests try to find the action name so let's try this ^

Reply:
Sounds good